requirements.txt

```diff
@@ -26,7 +26,7 @@ aiosmtpd
 marshmallow
 requests
 GitPython
-nbformat
+nbformat==5.7.0
```

Comment:
it might end up not working in some environments. It is sufficient for starting renku session.
Is it possible to support both versions?

Reply:
ok I will revert this change

for the `id`, it will simply try to pop it, trying to avoid the keyerror that would otherwise happen